database.js

```diff
@@ -17,10 +18,22 @@ function init() {
 
 // Make sure that the database schema is up to date
 function ensureVersion() {
-    const query = "SHOW COLUMNS FROM mentors LIKE 'pmsg'";
-    connection.query(query, (err, rows) => {
+    // Add the current version to the "version" table if it is empty
+    const checkVer = "SHOW COLUMNS FROM version LIKE 'ver'";
+    connection.query(checkVer, (err, rows) => {
         if (err) throw err;
         if (rows.length === 0) {
+            const addVer = "INSERT INTO version VALUES (?)";
+            connection.query(addVer, [package.version], (err) => {
+                if (err) throw err;
+            });
+        }
+    });
+    // Detect older versions of the database
+    const query = "SELECT ver FROM version";
+    connection.query(query, (err, rows) => {
+        if (err) throw err;
+        if (rows[0] === "1.0.2") {
```

Comment:
Does this return the most recently added entry? I thought `INSERT` appends to the end of a table, not the beginning.